meetings/agenda-2018-03-16.md

```diff
@@ -40,7 +40,7 @@ Contact Daniel Ehrenberg (littledan@igalia.com) for the link to the Google Hango
 1. New proposals
     1. [NumberFormat changes](https://github.com/tc39/ecma402/issues/215), including (a) restructuring the spec, (b) minor changes to behavior, (c) support for measure units, and (d) support for scientific and compact notation.
         - Status update for this proposal?
-        - How is the [C API](http://bugs.icu-project.org/trac/ticket/13597) coming (necessary for ChakraCore, at least)
+        - How is the [C API](https://ssl.icu-project.org/trac/ticket/13597) coming (necessary for ChakraCore, at least)
```

Comment:
https://unicode-org.atlassian.net/browse/ICU-.....

There are more.  You can do a global replace if you want to change them.

Reply:
sure thing!